cmd/exporter/exporter.go

```diff
@@ -20,6 +20,7 @@ import (
 	"flag"
 	"fmt"
 	"net/http"
+	prof "net/http/pprof"
```

Comment:
https://pkg.go.dev/net/http/pprof 
says `import _ "...."` and use the default mux.

Reply:
I had another look at https://go.dev/src/net/http/pprof/pprof.go . The overhead isn't much at all. Lets enable pprof by default.